File: src/test/testdata/annotators/colorAnnotator/JavaToKotlin.java
import com.badlogic.gdx.graphics.Color;
import com.badlogic.gdx.graphics.Colors;

public class JavaToKotlin {

  static final String c1 = "ff0000";
  static final String c2 = "#00ff00";
  static final Color cf3 = Color.valueOf("0000ffff");
  static final Color c4 = Color.valueOf(c2);
  static final Color c5 = Color.BLUE;
  static final Color c6 = Color.valueOf(c1);
  static final Color c7 = new Color(4456);
  static final Color c8 = new Color(0xff0000ff); static final Color sfdwe = c7;
  static final Color c9 = new Color(1, 0, 1, 1);
  static final Color c10 = Colors.get("foobar");
  static final Color c11 = Colors.getColors().get("some");

  public static class C {
    public final static String c1 = "ff0000";
  }

  void m() {
    Colors.put("foobar", Color.YELLOW);
    Colors.getColors().put("some", Color.RED);
  }

}

File: src/test/testdata/annotators/colorAnnotator/JavaToKotlin.kt
import com.badlogic.gdx.graphics.Color
import com.badlogic.gdx.graphics.Colors

val abc1 = <weak_warning descr="#708090ff">Color.SLATE</weak_warning>
val abcc2 = Color(<weak_warning descr="#708090ff">abc1</weak_warning>)
val abc3 =Color(Color(Color(<weak_warning descr="#ff0000ff">Color.valueOf(JavaToKotlin.c1)</weak_warning>)));
val abc4 = <weak_warning descr="#00ff00ff">JavaToKotlin.c2</weak_warning>;

val ab2 = <weak_warning descr="#00ff00ff">JavaToKotlin.c2</weak_warning>
val ab3 = <weak_warning descr="#0000ffff">JavaToKotlin.cf3</weak_warning>
val ab4 = <weak_warning descr="#00ff00ff">JavaToKotlin.c4</weak_warning>
val ab5 = <weak_warning descr="#0000ffff">JavaToKotlin.c5</weak_warning>
val ab6 = <weak_warning descr="#ff0000ff">JavaToKotlin.c6</weak_warning>
val ab7 = <weak_warning descr="#00001168">JavaToKotlin.c7</weak_warning>
val ab8 = <weak_warning descr="#ff0000ff">JavaToKotlin.c8</weak_warning>
val ab9 = <weak_warning descr="#ff00ffff">JavaToKotlin.c9</weak_warning>
val ab10 = <weak_warning descr="#ffff00ff">JavaToKotlin.c10</weak_warning>
val ab11 = <weak_warning descr="#ff0000ff">JavaToKotlin.c11</weak_warning>

val abffffk8 = <weak_warning descr="#ff0000ff">Color.valueOf(JavaToKotlin.C.c1)</weak_warning>;
val dffd = <weak_warning descr="#ffff00ff">Color(Color.RED.r, Color.RED.r, Color.RED.b, 1f)</weak_warning>